src/pages/journal/journal.ts

```diff
@@ -157,6 +164,13 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
     this.loadJournal();
   };
 
+  public formatLastRefreshTime() {
```

Comment:
You could pipe this function to the outcome of `getLastRefreshed` (which would be best to call `getLastRefreshedTime`) so that we don't need to call this function in the html template but just get the lastRefreshedTime$ asynchronously.

What do you think?

Reply:
Are you referring to storing strictly the formatted date string in the page state? While this would make the template cleaner, I think that a `Date` type in the page state is a better solution as this prevents dataloss being propagated into the state. The reason being, we have a story in the upcoming sprint to start using the last refresh datetime as a basis for automatic refresh - this would be tougher if we store a formatted string in the state.

Reply:
If I have misinterpreted the suggested solution, please let me know!

Reply:
After discussion, the proposed solution _is_ to simply keep a string of the formatted date in the journal page state. The reason for this is down to the intended usage of the page state, which should, wherever possible, be purely representational. I will refactor to adhere to this principle.